Directory.Build.props

```diff
@@ -113,4 +113,15 @@
 
   <Import Project="$(RepositoryEngineeringDir)Analyzers.props" />
 
+  <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
+     <_TimeZonesToBundle Include="africa" />
+     <_TimeZonesToBundle Include="antarctica" />
+     <_TimeZonesToBundle Include="asia" />
+     <_TimeZonesToBundle Include="australasia" />
+     <_TimeZonesToBundle Include="etcetera" />
+     <_TimeZonesToBundle Include="europe" />
+     <_TimeZonesToBundle Include="northamerica" />
+     <_TimeZonesToBundle Include="southamerica" />
+   </ItemGroup>
```

Comment:
This doesn't belong in the root Directory.Build.props which is imported in every project in the repository. Please consider moving this itemgroup into a props file that is imported in the places where this is needed.

Reply:
Hm this is needed in `eng/testing/tests.mobile.targets`, and `src/mono/wasm/wasm.targets`. We could add this to `src/mono/wasm/wasm.props`, and import that in  `wasm.targets`, and `eng/testing/tests.mobile.targets`. Thoughts?
@lewing @akoeplinger

Reply:
You could move this into its own props file, ie timezones.props which is then imported in those two consuming places.

Reply:
@tqiu8 could you please move that itemgroup to `src/mono/timezones.props`. And import that in `wasm.targets, and `tests.mobile.targets`?

In wasm.targets: `<Import Project="timezones.props" />`
In `tests.mobile.targets`, you will have to import it like:
`<Import Project="$(MonoProjectRoot)\wasm\timezones.props />`.

And make sure to do it before the item gets used by `CompileTimeZoneData`.